JoonasHeinonen/escape-from-zordoom-prison
# feature/test-level: Added model and the scene.

### Link to the ticket: [Test level](https://trello.com/c/SNAYeUv6/26-test-level)
![image](https://user-images.githubusercontent.com/33086486/190877211-857d70d4-3907-42c5-9adc-996872d9a235.png)

**Changes:**
 - Imported 3D model for the Paris blockhouse.
 - Created as scene of the blockhouse. Added custom collision to it.
 - Added it to the test level.
 - Added night light to the test level.
    - min energy: 0.01
    - max energy: 0.6

**How to test:**
 - Verify that the collision is consistent between player and the block house.
 - See that the light is good for the nightlife in Paris.
 
**Sources:**
 - [Godot 3 - 3D Custom Collision](https://www.youtube.com/watch?v=nh0iwfuO8AE&ab_channel=gameDevOutpost)
